tests/main/i18n/task.yaml

```diff
@@ -1,5 +1,9 @@
 summary: Test that i18n works
 
+details: |
+  This test cheks the translations do not causes crashes nor warnings.
```

Comment:
cheks => checks
the => that
causes => cause
crashes nor warnings => crashes or warnings

Reply:
updated

Reply:
Thanks!